admin/src/react-components/toolbar-without-delete.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Toolbar, SaveButton } from "react-admin";
+
+export const ToolbarWithoutDelete = props => {
```

Comment:
Clearly we should have done this a long time ago heh, nice and simple solution. I might have called this `SaveButtonToolbar` or `SaveOnlyToolbar`, but that's personal preference.

Reply:
Open to changing it -- I don't feel strongly -- though I think the name is clear for its purpose, I'd add a comment if I changed it. When we integrate delete functionality we only need to search for "Delete" where it's referenced.

Reply:
Yep I don't have strong preference either, just wanted to note.